### ISSUE TITLE
Support for multiple controllers in Boundary Desktop 

### ISSUE THREAD

**Issue by reza-solaris** [hashicorp/boundary#3157]
Hello,
Currently, if there are multiple Boundary installations, the user must sign out from one instance and connect to the other. The user must also type in the URL of each installation at every login.

It would be very beneficial if the Boundary Desktop client would allow multiple active instance connections. Something like having multiple instances of Boundary Desktop client open at the same time. To overcome this, we are currently using ```open -n -a Boundary.app``` to authenticate to multiple controllers at the same time. But it is not a convenient solution.

Something like this:
<img width="648" alt="Screenshot 2023-04-11 at 1 22 23 PM" src="https://user-images.githubusercontent.com/110919409/231147169-10ac5500-a2de-4ecc-9ded-325a4aa77756.png">

**Comment by xingluw** [comment]
Hi @reza-solaris, thank you for this feature request, this is something we have considered adding to the product, I will leave this post open to gauge community interest.

**Comment by reza-solaris** [comment]
Although it is possible to have all the targets from different environments in one controller, it would mean having a network connectivity from a single controller to other network segments (controller-worker connectivity). Thus, that would break the network segregation.

**Comment by reza-solaris** [comment]
Yes. Correct. The purpose of this functionality is to have production network complete segregation.

**Comment by muthukumars** [comment]
boundary authenticate to a controller1-url from command line
boundary authenticate to a controller2-url from command line.
one can access targets in controller1 as well as controller2  because both tokens are available and stored.
the same is not in Boundary desktop as per this posting.

Use cases could be possible
1. i do not want to allow the users to access targets  in controller1 when I am authenticated on controler2 and vice-versa (both in CLI and desktop) - looks like desktop works that way. how do we do in CLI
2. it is also possible use case, both CLI and boundary desktop may expect to deal with targets on both

Please suggest

**Comment by mekkim** [comment]
+1 for current client I'm working with who wants this supported in Boundary Desktop Clients for exactly the reasons of multiple control plane network isolation that @reza-solaris specifies.

**Comment by mekkim** [comment]
@xingluw Any updates on this?  Is it on the roadmap?
